### ISSUE TITLE
ModuleNotFoundError: No module named 'tdt.version'

### ISSUE THREAD

**Issue by jleseane23** [kquinsland/the-missing-todoist-tools#6]
My OS is Win 10. When I try to run `tmtdt.py --help` I get the following message.

`Traceback (most recent call last):
  File "C:\HTPC\Apps\the-missing-todoist-tools\tmtdt.py", line 3, in <module>
    import tdt.actions
  File "C:\HTPC\Apps\the-missing-todoist-tools\tdt\__init__.py", line 3, in <module>
    from tdt.version import __version__
ModuleNotFoundError: No module named 'tdt.version'`

- I tried in both the standard environment and the virtual environment using venv
- It was able to download all of the requirements necessary

**Comment by kquinsland** [comment]
Hi!

Thanks for being a windows tester!  After releasing this tool, covid and a new job, life started to eat _all_ of my time and I started to draw up a new architecture for a version of this tool that would solve a lot of the existing problems w/r/t  how kludgy it is to detect when a task has been completed. 

The error you have should be a simple fix. There's a file that has nothing but the version info in it and it _should_ have been created automatically by the github actions pipe.

Basically, you need to run https://github.com/kquinsland/the-missing-todoist-tools/blob/master/version-hook.sh

But as you're on windows, it'll probably be easier if you just make a file `tdt/version.py` with this content:


```python3
# Last modified: Tue Jul 13 17:08:05 PDT 2021
##
# the TDT version
__version__ = 'v0.0.1-8281e32-20200726'
```

**Comment by jleseane23** [comment]
Yup that worked.

Thanks for the help!
